### PR TITLE
Myrkiti Farm / Aqueduct Bridge Area

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

**Comment by terrapin47** [comment]
Please merge in https://github.com/Treeofsavior/EnglishTranslation/pull/1191 before merging this branch in.

File: ETC.tsv [hunk 1]
```diff
@@ -10737,8 +10737,8 @@ ETC_20150406_010738	{memo X}IvnItem:FARM47_1_SQ_060_ITEM_1:1:0/DoTimeAction:Maki
 ETC_20150406_010739	Create a magic circle with the wood Arunas gave you
 ETC_20150406_010740	{memo X}Burn the shaman doll in the magic circle
 ETC_20150406_010741	{memo X}Opening the alcohol container using the wooden hammer
-ETC_20150406_010742	Kill orange dandel and get kindling
-ETC_20150406_010743	Put the kindling in the burning magic circle and keep the fire alive
+ETC_20150406_010742	Kill Orange Dandel for kindling
```
Comment (Sunaries) [inline ETC.tsv]: Kill -> Defeat

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -8052,20 +8052,20 @@ QUEST_LV_0100_20150908_008047	Collimencia has appeared!
...
+QUEST_LV_0100_20150908_008054	The magic circle has been enhanced as you eliminated 4 crystals!
+QUEST_LV_0100_20150908_008055	Put the evil energy of the monsters into the pot which Rimas gave to you and bury it in front of the magic circle!
+QUEST_LV_0100_20150908_008056	You should collect the magical energy of the monsters into the pot and bury it in front of the magic circle!
 QUEST_LV_0100_20150908_008057	The attacks of the monsters begin!
-QUEST_LV_0100_20150908_008058	Looking at the magic field
-QUEST_LV_0100_20150908_008059	There is some strange vibration at the magic field
-QUEST_LV_0100_20150908_008060	Go back to the wizard, Rimas and report him about the magic field
+QUEST_LV_0100_20150908_008058	Looking at the magic circle
+QUEST_LV_0100_20150908_008059	There is some strange vibration at the magic circle
+QUEST_LV_0100_20150908_008060	Go back to the wizard, Rimas and report him about the magic circle
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Rimas -> Limas (assuming we're speaking of that magician)

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -8052,20 +8052,20 @@ QUEST_LV_0100_20150908_008047	Collimencia has appeared!
...
-QUEST_LV_0100_20150908_008052	Some strange crystal has appeared from the magic field{nl}Eliminate the strange crystal
-QUEST_LV_0100_20150908_008053	You can enhance the magic field only when you break the jewel that appeared nearby
-QUEST_LV_0100_20150908_008054	The magic field has been enhanced as you eliminated 4 crystals!
-QUEST_LV_0100_20150908_008055	Put the evil energy of the monsters into the pot which Rimas gave to you and bury it in front of the magic field!
-QUEST_LV_0100_20150908_008056	You should collect the magical energy of the monsters into the pot and bury it in front of the magic field!
+QUEST_LV_0100_20150908_008051	Looking at the magic circle
+QUEST_LV_0100_20150908_008052	Some strange crystal has appeared from the magic circle{nl}Eliminate the strange crystal
+QUEST_LV_0100_20150908_008053	You can enhance the magic circle only when you break the jewel that appeared nearby
+QUEST_LV_0100_20150908_008054	The magic circle has been enhanced as you eliminated 4 crystals!
+QUEST_LV_0100_20150908_008055	Put the evil energy of the monsters into the pot which Rimas gave to you and bury it in front of the magic circle!
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: Rimas -> Limas (assuming we're speaking of that magician)

**Comment by terrapin47** [comment]
@imcgames resolved conflicts with #1191, thanks.

**Comment by terrapin47** [comment]
oops, good to merge

**Comment by imcgames** [comment]
Thank you ^_^

**Pull request merged**